e2e/support/index.ts

```diff
@@ -2,3 +2,57 @@
  * Copyright Strimzi authors.
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
+import 'cypress-axe';
+import { Result } from 'axe-core';
+
+declare global {
+    namespace Cypress {
+      interface Chainable<Subject> {
+        logA11yViolations(violations: Result[], target: string): Chainable<Element>;
+        testA11y(target: string): Chainable<Element>;
+      }
+    }
+  }
+
+Cypress.Commands.add(
+  'logA11yViolations',
+  (violations: Result[], target: string) => {
+    // pluck specific keys to keep the table readable
+    const violationData = violations.map(
+      ({ id, impact, description, nodes }) => ({
+        id,
+        impact,
+        description,
+        nodes: nodes.length,
+      })
+    );
+    cy.log(
+      'Accessibility violation',
+      `${violations.length} accessibility violation${
+        violations.length === 1 ? '' : 's'
+      } ${violations.length === 1 ? 'was' : 'were'} detected ${
+        target ? `for ${target}` : ''
+      }`
+    );
+    cy.log('logTable', violationData);
+  }
+);
+
+Cypress.Commands.add('testA11y', (target: string) => {
+  cy.injectAxe();
```

Comment:
Can the `inject` and `configure` be run multiple times inside a test (expect there will be multiple calls to `testA11y` - or does it have to be ran only once per test?

Reply:
It only has to be run once.

Reply:
`cy.testally` would be called multiple times - so should `cy.injextAxe()` and `configureAxe()` instead be in a before all?

Reply:
yeah that makes sense I'll modify it.